web/src/components/Plugin/UI/fault-injection.tsx

```diff
@@ -0,0 +1,201 @@
+    span: 6,
+  },
+  wrapperCol: {
+    span: 7,
+  },
+};
+
+const FORM_LIST_LAYOUT = {
+  labelCol: {
+    span: 6,
+  },
+  wrapperCol: {
+    span: 10,
+  },
+};
+
+const FORM_ITEM_WITHOUT_LABEL = {
+  wrapperCol: {
+    sm: { span: 10, offset: 6 },
+  },
+};
+
+const FaultInjection: React.FC<Props> = ({ form }) => {
+  const { formatMessage } = useIntl()
+
+  return (
+    <Form
+      form={form}
+      {...FORM_ITEM_LAYOUT}
+    >
+      <Form.Item
+        name={['abort', 'http_status']}
+        label='abort.http_status'
+        required
+        tooltip={formatMessage({ id: "component.pluginForm.fault-injection.abort.http_status.tooltip" })}
+      >
+        <InputNumber min={200} required />
+      </Form.Item>
+      <Form.Item
+        name={['abort', 'percentage']}
+        label='abort.percentage'
+        tooltip={formatMessage({ id: "component.pluginForm.fault-injection.abort.percentage.tooltip" })}
+      >
+        <InputNumber min={0} max={100} />
+      </Form.Item>
+      <Form.Item
+        name={['abort', 'body']}
+        label='abort.body'
+        tooltip={formatMessage({ id: "component.pluginForm.fault-injection.abort.body.tooltip" })}
+      >
+        <Input min={200} />
+      </Form.Item>
+
+      <Form.List name={['abort', 'vars']}>
+        {(fields, { add, remove }) => {
+          return (
+            <div>
+              {fields.map((field, index) => (
+                <Form.Item
+                  {...(index === 0 ? FORM_LIST_LAYOUT : FORM_ITEM_WITHOUT_LABEL)}
+                  label={index === 0 && 'abort.vars'}
+                  tooltip={formatMessage({ id: "component.pluginForm.fault-injection.abort.vars.tooltip" })}
+                  key={field.key}
+                >
+                  <Form.Item
+                    {...field}
+                    validateTrigger={['onChange', 'onBlur']}
+                    noStyle
+                  >
+                    <Select style={{ width: '70%' }}>
+                      {['==', '`~=', '>', '<', '~~', '~*', 'in', 'has', '!'].map((item) => {
+                        return <Select.Option value={item} key={item}>{item}</Select.Option>
+                      })}
+                    </Select>
+                  </Form.Item>
+                  {fields.length > 1 ? (
+                    <MinusCircleOutlined
+                      className="dynamic-delete-button"
+                      style={{ margin: '0 8px' }}
+                      onClick={() => {
+                        remove(field.name);
+                      }}
+                    />
+                  ) : null}
+                </Form.Item>
+              ))}
+              {
+                <Form.Item {...FORM_ITEM_WITHOUT_LABEL}>
+                  <Button
+                    type="dashed"
+                    onClick={() => {
+                      add();
+                    }}
+                  >
+                    <PlusOutlined /> {formatMessage({ id: 'component.global.create' })}
```

Comment:
It would be better to 
```suggestion
                    <PlusOutlined /> `$({formatMessage({ id: 'component.global.create' })}) abort.vars`
```